### PR TITLE
changes for v1.1.0

### PR DESCRIPTION
**Pull request by jrvidal**
Update docs for the new 1.1.0 version. What to do about API reference changes is TBD. I want to upgrade the docs themselves to use 1.1.0, but we might need to ship a fix before.

### CODE REVIEW

File: docs/.vitepress/theme/components/Examples/WCEmbed/webcontainer.ts [hunk 1]
```diff
@@ -131,7 +131,7 @@ async function bootWebContainer(terminal: import('xterm').Terminal) {
 
   terminal.write('Booting WebContainer...');
 
-  webcontainer = WebContainer.boot();
+  webcontainer = WebContainer.boot({ workdirName: 'demo' });
```
Comment (jrvidal) [inline docs/.vitepress/theme/components/Examples/WCEmbed/webcontainer.ts:134]: :star: This changes the folder name visible in the terminal from `~/webcontainersio-98a7w9` or whatever to simply `~/demo`.

File: docs/changelog.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Changelog
+
+## 1.1.0
+
+* The value of the [`Cross-Origin-Embedder-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy) header
+  can be changed now with `BootOptions#coep`.
+* The name of the working directory can be specified through `BootOptions#workdirName`.
+* The default value of the `PATH` environment variable is exposed in `WebContainer#path`.
+* The full path of the working directory is exposed in `WebContainer#workdir`.
```
Comment (jrvidal) [inline docs/changelog.md:19]: :warning: These 3 items are minor stuff that I don't find particularly relevant for our tutorials.

**Comment by jrvidal** [comment]
Oh, lol, v1.1.0 broke some stuff, so we can't dogfood just yet. I'll revert for now.

File: docs/changelog.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - ['meta', {name: 'twitter:title', content: *title}]
+  - ['meta', {name: 'twitter:description', content: *description}]
+
+---
+
+# Changelog
+
+## 1.1.0
+
+* The value of the [`Cross-Origin-Embedder-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy) header
```
Comment (sylwiavargas) [inline docs/changelog.md:15]:
```suggestion
* The value of the [`Cross-Origin-Embedder-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy) header.
```

for consistency

File: docs/guides/configuring-headers.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ head:
 
 This guide provides an overview on how to configure COOP/COEP headers, including hosting platform configuration.
 
-WebContainers require that your page, even in development, be served with these two headers:
+WebContainers require that your page, even in development, is served with these two headers:
```
Comment (sylwiavargas) [inline docs/guides/configuring-headers.md:15]: good catch, thanks!

File: docs/changelog.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  can be changed now with `BootOptions#coep`.
```
Comment (sylwiavargas) [inline docs/changelog.md:16]:
```suggestion
* The value of the [`Cross-Origin-Embedder-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy) header can be changed now with `BootOptions#coep`.
```

^ removing the line break to improve readability in `.md` format

**Review by sylwiavargas — APPROVED** [review]
thanks for the additions and editions 💙

**Pull request merged**